src/components/Rsvp.tsx

```diff
@@ -0,0 +1,116 @@
+import React, { useCallback, useEffect, useMemo, useState } from 'react';
```

Comment:
[non-blocking]: Just for my understanding, cuz im a little confused, what's the difference between this file and the `pages/rsvp.tsx`? 🤔

Reply:
I copied all of the logic from `pages/rsvp.rsx`, which `nextjs` uses to create a route. For our use case, we want to render the RSVP form at the bottom of the home `/` route and use an `id` anchor. `pages/rsvp` should actually reroute to `/#rsvp` eventually.